data/json/monsters/zed_misc.json

```diff
@@ -1476,6 +1538,57 @@
     ],
     "armor": { "bash": 5, "cut": 3, "bullet": 2, "electric": 2 }
   },
+  {
+    "id": "mon_zombie_ocular",
+    "type": "MONSTER",
+    "name": { "str": "ocular zombie" },
+    "description": "This otherwise-regular zombie's head is absolutely filled to the brim with eyes, half of them frantically swiveling in their orbits.  The other half are staring directly at you.",
+    "default_faction": "zombie",
+    "species": [ "ZOMBIE", "HUMAN" ],
+    "volume": "62500 ml",
+    "weight": "81500 g",
+    "hp": 90,
+    "speed": 80,
+    "material": [ "flesh" ],
+    "symbol": "Z",
+    "color": "light_gray_white",
+    "scents_tracked": [ "sc_human", "sc_fetid" ],
+    "aggression": 100,
+    "morale": 100,
+    "melee_skill": 7,
+    "melee_dice": 2,
+    "melee_dice_sides": 6,
+    "melee_damage": [ { "damage_type": "cut", "amount": 1 } ],
+    "dodge": 7,
+    "bleed_rate": 125,
+    "vision_day": 1000,
+    "vision_night": 1000,
```

Comment:
This will have perfect vision of anything not obstructed between it and inside the reality bubble.  Even at night.  When does this thing start showing up in game?

Reply:
Turns out the answer to that question is nowhere because I didn't update the appropiate groups. Buh.

That said I was waffling on this in the PR body so I'll probably just limit their supervision to day.

Reply:
> This will have perfect vision of anything not obstructed between it and inside the reality bubble. Even at night. When does this thing start showing up in game?

Did limit their vision as I said. I also went ahead and actually checked ingame to see spawn rates, and they seem good.